docs/architecture/update-db-urls.md

```diff
@@ -0,0 +1,29 @@
+# Update the URLs in the content
```

Comment:
@mburridge I'm reviewing this docs and I don't really understand the scenario where this URL replacement is needed.

In the text it says "if you're using Frontity in Decoupled Mode and are migrating an existing WordPress site to Frontity...."

As I see it, the most common scenario for migrating a WP site to Frontity will be something like...

if I have a WP site under the domain `www.juanmaguitar.com` and I want to migrate this WP site to Frontity I can do the following:
- Connect the domain `www.juanmaguitar.com` to the NodeJs server hosting the Frontity app
- Connect a new subdomain `wp.juanmaguitar.com` to the WordPress server

But in this scenario, the WordPress server `wp.juanmaguitar.com` will already have all the links inside pointing to  `www.juanmaguitar.com` so, there would be no need of URL replacements in this case, right?

Can you clarify with examples (domain and subdomain) the scenarios where a URL replacement is needed?

Reply:
@juanmaguitar I think problems could arise with links in new content added after the WordPress URL settings are changed.

Using your example links in content before the change will, as you observe, already point to `www.juanmaguitar.com`. But links in content added after the change will point to `wp.juanmaguitar.com`, as WP will use the URL setting to determine where the internal links should point to.

So you will eventually have two different domains being pointed to in your content for internal links.

Also the [link processor](https://api.frontity.org/frontity-packages/collections-packages/components#the-link-processor) uses the same domain as the WordPress data source to convert links to a `<Link />` component. So if the internal links are left pointng to `www.juanmaguitar.com` the link processor won't work on those links because it will only convert links pointing to `wp.juanmaguitar.com`, i.e. the URL of the WordPress data source.

Hope that all makes sense.

Reply:
Understood!! Thanks @mburridge